uint32 frame_rate
uint8  tag
uint16 status
uint8 ins_status
uint16 gpst_wn   
uint32 gpst_tow
int16 gyr_b_x
int16 gyr_b_y
int16 gyr_b_z
int16 acc_b_x
int16 acc_b_y
int16 acc_b_z
int16 mag_b_x
int16 mag_b_y
int16 mag_b_z
int16 air_pressure
uint8 reserved_1
uint8 reserved_2
int8 temperature
uint8 utc_year
uint8 utc_mouth
uint8 utc_day
uint8 utc_hour
uint8 utc_min
uint16 utc_sec
int16 roll
int16 pitch
uint16 yaw
int16 quat_w
int16 quat_x
int16 quat_y
int16 quat_z
int32 ins_lon
int32 ins_lat
int32 ins_msl
uint8 pdop
uint8 hdop
uint8 solq_pos
uint8 nv_pos
uint8 solq_heading
uint8 nv_heading
uint8 diff_age
int16 undulation
uint8 reserved_3
int16 vel_enu_1
int16 vel_enu_2
int16 vel_enu_3
int16 acc_enu_1
int16 acc_enu_2
int16 acc_enu_3
uint32 reserved_4
uint32 reserved_5
uint32 reserved_6
uint16 reserved_7